src/js/client/payment.js

```diff
@@ -27,13 +27,13 @@ module.exports = function () {
             method: 'POST',
             data: data
           })
-            .success((data) => {
-              $('.payment-form').remove();
-              $('.payment-form-description').text(data.message);
+            .done((data) => {
+              $output.text(data.message);
```

Comment:
If the first payment fails, the user retries the payment, and it succeeds the second time, we should remove the red class from the status section on the success.